src/main/java/domain/AttendTime.java

```diff
@@ -1,23 +1,52 @@
 package domain;
 
-import java.time.LocalDateTime;
-import java.time.format.DateTimeFormatter;
+import java.time.LocalDate;
+import java.time.LocalTime;
 
 public class AttendTime {
 
-    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+    private final LocalDate localDate;
+    private final LocalTime localTime;
+    public AttendTime(LocalDate localDate,LocalTime localTime) {
+        validate(localDate);
+      this.localDate=localDate;
+      this.localTime=localTime;
+    }
+
+    public void validate(LocalDate localDate) {
+        validateWeekend(localDate);
+        validateHoliday(localDate);
+    }
+
+    private void validateWeekend(LocalDate localDate) {
+        if (localDate.getDayOfWeek().getValue() > 5)
+            throw new IllegalArgumentException("주말은 출석 할 수 없습니다.");
+    }
```

Comment:
validateHoliday 메서드처럼 만들어둔 December 객체를 활용해보면 숫자 5보다 주말을 잘 표현할 수 있을 것 같아요 😄 
```suggestion
    private void validateWeekend(LocalDate localDate) {
        if (December.isWeekend(localDate))
            throw new IllegalArgumentException("주말은 출석 할 수 없습니다.");
    }
```

Reply:
상수에 대한 부분을 놓쳤군요!

수정했습니다!

꼼꼼하게 봐주셔서 감사합니다! 😊